Library/Homebrew/brew.sh

```diff
@@ -402,15 +409,17 @@ Minimum required version: ${HOMEBREW_MINIMUM_CURL_VERSION}
 Your curl version: ${curl_name_and_version##* }
 Your curl executable: $(type -p $HOMEBREW_CURL)"
 
-    if [[ -z $HOMEBREW_CURL_PATH || -z $HOMEBREW_DEVELOPER ]]; then
+    if [[ -n $HOMEBREW_CURL_PATH && -z $HOMEBREW_DEVELOPER ]]
```

Comment:
```suggestion
    if [[ -n $HOMEBREW_CURL_PATH ]]
```

I think a manually specified `curl` path should always fail if it's too old.